#!/usr/bin/env python
PACKAGE = "my_pcl_tutorial"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("MIN_X", double_t, 0, "MIN_X",    -.5, -5,  10)
gen.add("MIN_Y", double_t, 0, "MIN_Y",    -.5, -5,  10)
gen.add("MIN_Z", double_t, 0, "MIN_Z",    -.5, -5,  10)

gen.add("MAX_X", double_t, 0, "MAX_X",    .5, 0,   10)
gen.add("MAX_Y", double_t, 0, "MAX_Y",    .5, 0,   10)
gen.add("MAX_Z", double_t, 0, "MAX_Z",    .5, 0,   10)

exit(gen.generate(PACKAGE, "my_pcl_tutorial", "CropBox"))
